subtests/docker_cli/run/run_tmpfs.py

```diff
@@ -0,0 +1,56 @@
+from run import run_base
+from dockertest.dockercmd import DockerCmd
+from dockertest.output import OutputGood
+from dockertest.output import mustpass
+from dockertest.output import DockerVersion
+from dockertest import xceptions
+
+
+class run_tmpfs(run_base):
+    # Verify behavior when multiple --name options passed
+
+    def initialize(self):
+        super(run_tmpfs, self).initialize()
+        # On docker 1.10, the second tag should pass. On < 1.10, fail.
+        expected_version = self.config['min_docker_ver']
+        try:
```

Comment:
You might be overthinking this :-). All you need is:

    DockerVersion().require_server(self.config['min_docker_ver'])

No need for the try/except/failif_ne